sdk/src/main/java/dev/cdevents/constants/CDEventConstants.java

```diff
@@ -1,22 +1,42 @@
 package dev.cdevents.constants;
 
+import dev.cdevents.events.*;
```

Comment:
Maybe a comment on why the import ...*  i.e we need all the events

Reply:
Linting doesn't stop this